sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
 # Release History
 
-## 2.0.0-beta.4 (Unreleased)
-
+## 2.0.0-beta.4 (2020-02-10)
+### Breaking changes
+- `CloudEvent` constructor now accepts parameter "data". Removed `setData()`.
+- `CloudEvent.parse()` and `EventGridEvent.parse()` are renamed to `fromString()`.
+- `CloudEvent::getData()` of CloudEvent and EventGridEvent now returns `com.azure.core.util.BinaryData`. 
+  Users can use methods `BinaryData` to deserialize event data. The generic version of `getData()` is then removed.
+- Removed `CloudEvent::getDataAsync()`
+- Added `EventGridSasGenerator` class and removed `EventGridSasCredential`. Use `EventGridSasGenerator.generateSas()` to
```

Comment:
The generateSas method will be moved to the client right? Will that be done in the next release?

Reply:
Yes, next release.